src/ui/public/courier/data_source/build_query/from_kuery.js

```diff
@@ -1,13 +1,25 @@
-import _ from 'lodash';
 import { fromKueryExpression, fromKqlExpression, toElasticsearchQuery, nodeTypes } from '../../../kuery';
+import { documentationLinks } from '../../../documentation_links';
 
-export function buildQueryFromKuery(indexPattern, queries) {
-  const queryASTs = _.map(queries, query => fromKueryExpression(query.query));
-  return buildQuery(indexPattern, queryASTs);
-}
+const queryDocs = documentationLinks.query;
 
-export function buildQueryFromKql(indexPattern, queries) {
-  const queryASTs = _.map(queries, query => fromKqlExpression(query.query));
+export function buildQueryFromKuery(indexPattern, queries = []) {
+  const queryASTs = queries.map((query) => {
+    try {
+      return fromKqlExpression(query.query);
```

Comment:
It's a bit hard to follow the Kql vs Kuery.  Here `buildQueryFromKuery` really means build query from either kuery or KQL.  But sometimes it's specific to only the language noted - fromKqlExpression will throw an error if the query is kuery, not Kql.  And fromKqlExpression is inside a folder named kuery.

Just giving my perspective as a new reader to this code.  Might be easier if the kql vs kuery differences could all be hidden inside the kuery folder and not exported.  Then only one name would need to be used externally.